README.md

```diff
@@ -250,7 +250,7 @@ Other Style Guides
 
   - [3.7](#3.7) <a name='3.7'></a> Group your shorthand properties at the beginning of your object declaration.
 
-  > Why? It's easier to tell which properties are using the shorthand.
+  > Why? It is easier to tell which properties are using the shorthand.
```

Comment:
It's because some places are using "Why? It's.." while other places are using "Why? It is.." and I thought making it consistent in one way would be best.